docs/narr/viewconfig.rst

```diff
@@ -494,6 +494,16 @@ configured view.
 
   .. versionadded:: 1.4a3
 
+``is_authenticated``
+
+  This value, if specified, must be either ``True`` or ``False``.  If it is
+  specified and is ``True``, the request must be for an authenticated user,
+  as determined by the :term:`security policy` in use.  If it is specified and
+  ``False``, the associated view callable will be invoked only if the request
+  does not have an authenticated user.
```

Comment:
If docs are incorrect, copying them is not the best option. In this case, it looks like the original author did not account for multiple predicates on a view or route config. I'd prefer to update the copy to make it correct. Would you please do so per [your comment](https://github.com/Pylons/pyramid/pull/3598#discussion_r449010244)?

Ideally the original docs from which the concept was copied would also get corrected, or at least an issue opened or amended. See https://github.com/Pylons/pyramid/issues/3597